### ISSUE TITLE
Modifying the pie shell only stacks with other modified pie shells

### ISSUE THREAD

**Issue by l1ving** [anegostudios/VintageStory-Issues#1727]
**Game Version:** 1.16.3
**Platform:** **Windows** / Mac / **Linux** (which?)   
**Modded:** **No** / Yes (which?)  
**SP/MP:** **Singleplayer** / **Multiplayer** (which?)  

### Description
Explain the issue you're running into.

The title might not be entirely clear, so I'll give some examples
- Unmodified plain crust type can stack with unmodified plain crust type
- Modified cross crust type can stack with modified cross crust type
- Modified plain crust type **cannot** stack with unmodified plain crust type
- Modified plain crust type can stack with modified plain crust type

TLDR: once you modify the crust type, even if you change it back to plain it can only stack with another pie that has been changed back to plain, and not one that was just created and not modified.

After you bake and slice up the pie, same-crust types are able to be stacked regardless of their modified status. No idea if it's the baking or the cutting step which resets this, I didn't get to check.

**Comment by DArkHekRoMaNT** [comment]
Confirmed in 1.18.5

**Comment by DArkHekRoMaNT** [comment]
Also pies don't display edi for some reason

**Comment by Craluminum2413** [comment]
Does it still happen in 1.21? There is a new crust type
